plugins/upstash/api_key_test.go

```diff
@@ -0,0 +1,59 @@
+package upstash
```

Comment:
This file triggers a few lint errors, let's look into those 😄

Reply:
No longer seems to be an issue, marking as resolved.